docs/sources/helm-charts/mimir-distributed/release-notes/v5.8.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Grafana Mimir Helm chart version 5.8 release notes"
+menuTitle: "V5.8 release notes"
+description: "Release notes for Grafana Mimir Helm chart version 5.8"
+weight: 300
+---
+
+# Grafana Mimir Helm chart version 5.8 release notes
+
+Grafana Labs is excited to announce version 5.8 of the Grafana Mimir Helm chart, which is compatible with Grafana Mimir v2.17 and Grafana Enterprise Metrics (GEM) v2.17. The `mimir-distributed` Helm chart is the best way to install Mimir on Kubernetes.
+
+The highlights that follow include the top features, enhancements, and bug fixes in this release. For a comprehensive list of changes, see the [Helm chart changelog](https://github.com/grafana/mimir/tree/main/operations/helm/charts/mimir-distributed/CHANGELOG.md).
+
+## Important changes
+
+KEDA autoscaling has changed `toPromQLLabelSelector` from an object to a list of strings, adding support for all PromQL operators.
+
+The Memcached timeout for the ruler-storage cache has increased from 200 milliseconds to 500 milliseconds.
+
+The environment variable `JAEGER_REPORTER_MAX_QUEUE_SIZE` is no longer set by default. Components use OTel's default value of 2048 unless explicitly configured. You can still configure `JAEGER_REPORTER_MAX_QUEUE_SIZE` if you configure tracing using Jaeger environment variables, and you can always set `OTEL_BSP_MAX_QUEUE_SIZE` for OTel configuration.
```

Comment:
this is part of a bigger change which is that mimir is now exporting otel traces instead of jaeger traces. I would present it as that - something along the lines of "Export OTEL traces instead of Jaeger: ...". Then maybe you can also point to the new documentation Oleg udpated about tracing; not sure?